Formula/f/flarectl.rb

```diff
@@ -1,6 +1,7 @@
 class Flarectl < Formula
   desc "CLI application for interacting with a Cloudflare account"
   homepage "https://github.com/cloudflare/cloudflare-go/tree/master/cmd/flarectl"
+  # Pin to 0.93.0 as cloudflare-go v2 does not produce flarectl
```

Comment:
This really sounds like it should be deprecated since flarectl is not supported anymore. Or will there be 0.x releases to fix bugs?

Reply:
there will be more in the 0.x release line while we transition other tools (mainly terraform) to the newer 2.x version.

Reply:
Maybe this should just have an adjusted livecheck then?

Reply:
Agreed but let's merge to stop auto bumps for now.